classes/form/CustomerFormatter.php

```diff
@@ -247,7 +247,7 @@ public function getFormat()
         return $this->addConstraints($format);
     }
 
-    private function addConstraints(array $format)
+    protected function addConstraints(array $format)
```

Comment:
same here, this function is not supposed to be called by your user, but in your own implementation you can do everything you want + and have to maintain it +. The main key is here, and a very important shift compared to 1.6- history

Reply:
what you mean "your own implementation"? if you need to do custom registration form with altered fields, username as a login and custom fields with custom validation what you would do?

Reply:
@mickaelandrieu why should i do a copy/paste? I just want to use this identical method in my override! Why should i maintaining myself something that is already maintained? This is just a waste of time...